exec/java-exec/src/main/java/org/apache/drill/exec/ops/OperatorMetricRegistry.java

```diff
@@ -61,14 +66,11 @@
 
   private static void register(final int operatorType, final Class<? extends MetricDef> metricDef) {
     // Currently registers a metric def that has enum constants
-    final MetricDef[] enumConstants = metricDef.getEnumConstants();
-    if (enumConstants != null) {
-      final String[] names = new String[enumConstants.length];
-      for (int i = 0; i < enumConstants.length; i++) {
-        names[i] = enumConstants[i].name();
-      }
-      OPERATOR_METRICS[operatorType] = names;
-    }
+    String[] names = Arrays.stream(metricDef.getEnumConstants())
```

Comment:
Do you know the case when enum constants are null?

Reply:
My bad, not right `null` check. EnumConstants array should be checked.